module/VuFindConsole/src/VuFindConsole/Command/Util/SwitchDbHashCommand.php

```diff
@@ -145,6 +159,26 @@ protected function getOpenSsl($algorithm)
         return ($algorithm == 'none') ? null : new Openssl(compact('algorithm'));
     }
 
+    /**
+     * Re-encrypt a row.
+     *
+     * @param UserRow|UserCardRow $row       Row to update
+     * @param ?BlockCipher        $oldcipher Old cipher (null for none)
+     * @param BlockCipher         $newcipher New cipher
+     *
+     * @return void
+     * @throws InvalidArgumentException
+     */
+    protected function fixRow($row, ?BlockCipher $oldcipher, BlockCipher $newcipher): void
+    {
+        $pass = ($oldcipher && $row['cat_pass_enc'] ?? null !== null)
```

Comment:
`&&` has a higher precedence than `??` so needs parentheses around `$row['cat_pass_enc'] ?? null`, right?

Reply:
Or do we need null coalescing at all? The column always exists even if null.

Reply:
It really is!